{"uid":"fa4ca6b012f0c12b","name":"test_default_sign_up_website_qr_type[-Default-website_qr_create]","fullName":"tests.tests_default_flow.test_default_sign_up_png_download_welcome_modal.TestDefaultSignUpFlowPng#test_default_sign_up_website_qr_type","historyId":"74c5b18cec54b2811074f7283074c21c","time":{"start":1735196880623,"stop":1735196880623,"duration":0},"status":"broken","statusMessage":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","statusTrace":"request = <SubRequest 'sign_up_fixture' for <Function test_default_sign_up_website_qr_type[-Default-website_qr_create]>>\nfake_email = 'wtl-automation7373387745627@test.com'\n\n    @pytest.fixture(scope=\"function\", params=os.getenv(\"LANGUAGES\", \"\").split(\",\"))\n    @allure.title(\"Sign up\")\n    def sign_up_fixture(request, fake_email):\n        language = request.param\n        base_url = f\"{ENV_URL}{language}/\"\n        email = fake_email\n>       request.instance.main_page.open_page(f\"{base_url}register/\")\n\nconftest.py:99: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:10: in open_page\n    self.page.goto(url)\n/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py:9002: in goto\n    self._sync(\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_page.py:553: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7f2de093f9e0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7f2de07f1620>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nE           Call log:\nE           navigating to \"None/register/\", waiting until \"load\"\n\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:520: Error","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":21,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1735196871910,"stop":1735196872010,"duration":100},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1735196872010,"stop":1735196872010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1735196872010,"stop":1735196872010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1735196872011,"stop":1735196872011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Set up browser: chromium","time":{"start":1735196872011,"stop":1735196873048,"duration":1037},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1735196872011,"stop":1735196872011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"context","time":{"start":1735196880624,"stop":1735196880652,"duration":28},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Failed test artifacts","time":{"start":1735196880624,"stop":1735196880624,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test","time":{"start":1735196880624,"stop":1735196880624,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"fake_email","time":{"start":1735196880624,"stop":1735196880624,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"page","time":{"start":1735196880652,"stop":1735196880763,"duration":111},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Sign up","time":{"start":1735196880763,"stop":1735196880882,"duration":119},"status":"broken","statusMessage":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/app/workspace/conftest.py\", line 99, in sign_up_fixture\n    request.instance.main_page.open_page(f\"{base_url}register/\")\n  File \"/app/workspace/base/base_page.py\", line 10, in open_page\n    self.page.goto(url)\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py\", line 9002, in goto\n    self._sync(\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_page.py\", line 553, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py\", line 145, in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 520, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"setup","time":{"start":1735196880763,"stop":1735196880763,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"page::0","time":{"start":1735196881088,"stop":1735196881090,"duration":2},"status":"broken","statusMessage":"AttributeError: 'Function' object has no attribute 'rep_call'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 52, in page\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"context::0","time":{"start":1735196881091,"stop":1735196881231,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test::0","time":{"start":1735196881232,"stop":1735196881232,"duration":0},"status":"broken","statusMessage":"requests.exceptions.MissingSchema: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 127, in delete_user_after_test\n    response = requests.post(DELETE_USER_URL, headers=headers, json=data)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/sessions.py\", line 575, in request\n    prep = self.prepare_request(req)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/sessions.py\", line 484, in prepare_request\n    p.prepare(\n  File \"/usr/local/lib/python3.12/dist-packages/requests/models.py\", line 367, in prepare\n    self.prepare_url(url, params)\n  File \"/usr/local/lib/python3.12/dist-packages/requests/models.py\", line 438, in prepare_url\n    raise MissingSchema(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Failed test artifacts::0","time":{"start":1735196881235,"stop":1735196881236,"duration":1},"status":"broken","statusMessage":"AttributeError: 'Function' object has no attribute 'rep_call'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 72, in artifacts\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Set up browser: chromium::0","time":{"start":1735196938808,"stop":1735196938901,"duration":93},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Default Sign Up Flow - PNG Format - chromium"},{"name":"tag","value":"smoke"},{"name":"parentSuite","value":"tests.tests_default_flow"},{"name":"suite","value":"test_default_sign_up_png_download_welcome_modal"},{"name":"subSuite","value":"TestDefaultSignUpFlowPng"},{"name":"host","value":"bd3659fb8bce"},{"name":"thread","value":"34-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.tests_default_flow.test_default_sign_up_png_download_welcome_modal"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"qr_create_method","value":"'website_qr_create'"},{"name":"resolution","value":"'Default'"},{"name":"sign_up_fixture","value":"''"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4d83cb380828f293","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196842842,"stop":1735196842842,"duration":0}},{"uid":"eaf0276c43366ec8","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196795494,"stop":1735196795494,"duration":0}},{"uid":"7eea53dea5f4bfdc","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196741287,"stop":1735196741287,"duration":0}},{"uid":"389795115fb42148","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196740163,"stop":1735196740163,"duration":0}},{"uid":"8177aae6fc6cc880","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196708534,"stop":1735196708534,"duration":0}},{"uid":"f2e3e885931a9531","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196662622,"stop":1735196662622,"duration":0}},{"uid":"fd114846fced0686","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196601737,"stop":1735196601737,"duration":0}},{"uid":"6a43d24710a67581","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196572442,"stop":1735196572442,"duration":0}},{"uid":"4a45254482671872","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196554244,"stop":1735196554244,"duration":0}},{"uid":"ed4924c3a7b1473a","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196530470,"stop":1735196530470,"duration":0}},{"uid":"3f9ed23e9f5d9ac4","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196457619,"stop":1735196457619,"duration":0}},{"uid":"795d3231a64b4a5f","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196433260,"stop":1735196433260,"duration":0}},{"uid":"4ae3850a17f805a2","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196393193,"stop":1735196393193,"duration":0}},{"uid":"f200e22a2a12e698","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196317220,"stop":1735196317220,"duration":0}},{"uid":"dc85be1c91f58088","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196255083,"stop":1735196255083,"duration":0}},{"uid":"f1291e92fde90810","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196251404,"stop":1735196251404,"duration":0}},{"uid":"4196ea93ddabd7f9","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196248516,"stop":1735196248516,"duration":0}},{"uid":"dfcc5f467873b728","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196127645,"stop":1735196127645,"duration":0}},{"uid":"598789187a5480cd","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196118942,"stop":1735196118942,"duration":0}},{"uid":"7b53d3e8035d3c1e","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196118923,"stop":1735196118923,"duration":0}},{"uid":"60331ee77a23b6c3","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196116149,"stop":1735196116149,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["smoke"]},"source":"fa4ca6b012f0c12b.json","parameterValues":["'website_qr_create'","'Default'","''"]}